format_spec/history.md

```diff
@@ -0,0 +1,170 @@
+---
+title: Format version history
+---
+
+# Format Version History
+
+## Version 22
+
+Introduced in TileDB 2.25
+
+* The _Current domain_ field was added to [array schemas](./array_schema.md#array-schema-file).
+
+## Version 21
+
+Introduced in TileDB 2.19
+
+* The TileDB implementation has been updated to fix computing [tile metadata](./fragment.md#tile-mins-maxes) for nullable fixed-size strings on dense arrays.
+
+> [!NOTE]
+> This version does not contain any changes to the storage format, but was introduced as an indicator for implementations to not rely on tile metadata under certain circumstances on previous versions.
```

Comment:
This only applies to fixed sized strings on dense arrays I think?

Reply:
I'm mostly worried someone reads this "not rely on tile metadata under certain circumstances on previous versions." and makes the wrong assumption. I think we should repeat for 100% clarity. This is the last issue for me in this PR, once we resolve it's good to go IMO.

Reply:
Updated.